fork-clone-workflow.md

```diff
@@ -0,0 +1,41 @@
+# Simple fork/clone development workflow
+
+A typical fork/clone development workflow is described here. This information is intended 
+for someone new to open source development and git source code control. The fork/clone
+development workflow is not the only workflow possible. In some situations developers
+do not fork the organizational repository but rather clone the organizational repository
+directly. In this document we address only the fork/clone development workflow, not the
+workflow in which forks are not used.
+
+## Assumptions
```

Comment:
I find this very useful and i think others will as well, reading this list they can feel more confident when proceeding and know the assumptions you have made when writing